source/applications/advanced/gamma_correction.py

```diff
@@ -58,9 +58,12 @@ def _capture_2d_image(camera):
     """
     settings_2d = zivid.Settings2D(acquisitions=[zivid.Settings2D.Acquisition()],)
     with camera.capture(settings_2d) as frame_2d:
-        image = frame_2d.image()
-        image_array = image.to_array()
-        return np.dstack([image_array["b"], image_array["g"], image_array["r"]])
+        image = frame_2d.image_rgba()
+        rgba = image.copy_data()
+        bgra = rgba.copy()
+        bgra[:,:,0] = rgba[:,:,2]
+        bgra[:,:,2] = rgba[:,:,0]
+        return bgra
```

Comment:
Why not just do `bgr = cv2.cvtColor(rgba, cv2.COLOR_RGBA2BGR)` ?

Reply:
Done, have to test also.

Reply:
Looks fine.